AsyncDisplayKit/ASPagerFlowLayout.m

```diff
@@ -54,6 +56,11 @@ - (NSIndexPath *)_indexPathForVisiblyCenteredItem
   return nil;
 }
 
+- (BOOL)_dataSourceIsEmpty
+{
+  return ([self.collectionView numberOfSections] > 0 && [self.collectionView numberOfItemsInSection:0] > 0);
```

Comment:
LOL thanks for catching that. Was an on the fly edit to the review. Need to fix it.